nodeEditor/src/diagram/graphCanvas.tsx

```diff
@@ -233,14 +233,18 @@ export class GraphCanvasComponent extends React.Component<IGraphCanvasComponentP
         }, false);     
 
         // Store additional data to serialization object
-        this.props.globalState.storeEditorData = (editorData) => {
-            editorData.zoom = this.zoom;
+        this.props.globalState.storeEditorData = (editorData, graphFrame) => {
             editorData.x = this.x;

```

Comment:
editorData.x and y and zoom should NOT be in if we re exporting a frame (look at the exported json. Everything not frame related should not be in)